test/parallel/test-fs-promises-read-file.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const common = require('../common');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const { createHash } = require('crypto');
+const { createReadStream } = require('fs');
+const { readFile } = require('fs/promises');
+const fixtures = require('../common/fixtures');
+
+const fn = fixtures.path('../../AUTHORS');
```

Comment:
This seems to defeat the purpose of `fixtures`. Maybe just create a file with

```js
const tmpdir = require('../common/tmpdir');
tmpdir.refresh();
const fn = path.join(tmpdir.path, 'large-file');
// Write a file and then read it again using `readFile`
```

Or, create a file with a known hash in fixtures.

Reply:
Good idea, this way I don't need the `crypto` module anymore for the test